community/cypher/spec-suite-tools/src/main/scala/cypher/cucumber/BlacklistPlugin.scala

```diff
@@ -30,7 +30,11 @@ import scala.io.Source
 object BlacklistPlugin {
   private var _blacklist: Set[String] = null
 
-  def blacklist(): Set[String] = {
+  def blacklisted(name: String) = blacklist().contains(normalizedScenarioName(name))
+
+  def normalizedScenarioName(name: String) = name.trim.toLowerCase
+
+  private def blacklist(): Set[String] = {
```

Comment:
If this is private, we might as well skip it and use the field directly. The assertion failing is just as helpful as the resulting NPE.